src/ontology/mondo-edit.obo

```diff
@@ -518029,14 +518031,11 @@ is_a: MONDO:0003847 {source="OMIM:619318"} ! hereditary disease
 id: MONDO:0036192
 name: obsolete EN1-related dorsoventral syndrome
 subset: ordo_disorder {source="Orphanet:611223"}
```

Comment:
@yshwetar this subset annotation will be updated automatically via the subset update pipeline. However, a new QC check will fail (we have a new QC that ensures that all the ordo subsets have an exact match to the orphanet source, this PR hasn't failed yet because the QC check hasn't been merged). 
Please remove this subset annotation in anticipation of this new QC check.

Reply:
Comment addressed